install/upgrade/update_to_dev.php

```diff
@@ -209,6 +209,11 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
       $migration->addPostQuery("DELETE FROM `$table` WHERE `symbol` IN ('" . $policiesStr . "')");
    }
 
+   // Add policies categories
+   $query = $query = "INSERT INTO `".PluginFlyvemdmPolicyCategory::getTable()."` VALUES (11,'Configuration',0,'Configuration',NULL,1,NULL,'[]'),
+      (12,'User interface',11,'Configuration > User interface',NULL,2,NULL,'{\"11\":\"11\"}')";
+   $DB->query($query) or plugin_flyvemdm_upgrade_error($migration);
+
```

Comment:
this is not needed. The code which creates the policies will generate the categories.

Reply:
FYI: https://github.com/flyve-mdm/glpi-plugin/blob/develop/install/installer.class.php#L329